### ISSUE TITLE
slice operations on pm_Matrix do not preserve the type

### ISSUE THREAD

**Issue by kalmarek** [oscar-system/Polymake.jl#130]
```julia
A = pm_Matrix{pm_Integer}([1 1 0; 1 0 1]);
@show typeof(A)
@show typeof(A[:, 2:end])
```
```
typeof(A) = Polymake.pm_MatrixAllocated{pm_Integer}
typeof(A[:, 2:end]) = Array{pm_Integer,2}
```

**Comment by sebasguts** [comment]
It is the same problem as in #129, I think. I will fix this.

**Comment by kalmarek** [comment]
fixed by #133
